code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -432,7 +432,23 @@
 
 //Overriding this will stop a number of headaches down the track.
 /mob/living/silicon/pai/attackby(obj/item/weapon/W as obj, mob/user as mob, params)
-	if(W.force)
+	if(istype(W, /obj/item/stack/nanopaste))
+	var/obj/item/stack/nanopaste/N = W

```

Comment:
You'll want to tab this one in one more level of indentation, as it should only typecast after the if check passes.